### DISCUSSION TITLE
Berkeley release 68fa3fa

### DISCUSSION THREAD

**Discussion by deepthiskumar** [MinaProtocol/mina#12905] [Berkeley]
Mina's ecosystem partner, O(1) Labs, is continuously working on improvements to the core protocol and today we are excited to announce an updated unstable release 68fa3fa that is ready to test on the Berkeley testnet! Please report any issues in the comments on this post or tag any issues you create with berkeley-rampup and include berkeley 68fa3fa in the issue title.

Improvements/features:
* Fix for bootstrap failure https://github.com/MinaProtocol/mina/pull/12808
* Fixed https://github.com/MinaProtocol/mina/issues/12809

No archive DB schema changes. Users can continue using the following schema files from the [previous release](https://github.com/MinaProtocol/mina/discussions/12757) and only update the archive process.

https://github.com/MinaProtocol/mina/blob/ae78a64a1ede801cbab748325566135f046993ff/src/app/archive/create_schema.sql
https://github.com/MinaProtocol/mina/blob/ae78a64a1ede801cbab748325566135f046993ff/src/app/archive/zkapp_tables.sql
Note: zkApps Test Transaction CLI tool cannot be used to create token accounts in this release.

Complete Changelog since ae78a64 in:
Github Compare: https://github.com/MinaProtocol/mina/compare/ae78a64a1ede801cbab748325566135f046993ff...68fa3faa1af5ab5ee4510ffe9d677a294dd96229

SnarkyJS change log: https://github.com/o1-labs/snarkyjs/blob/main/CHANGELOG.md
Upgrading & Connecting to Berkeley Testnet
Debian Packages:
Debian Repository:
First, set up and update the unstable Debian Repository for your platform. Replace the word CODENAME with the appropriate codename for your machine, one of bionic, focal, stretch, buster, or bullseye and run:

echo "deb [trusted=yes] http://packages.o1test.net/ CODENAME unstable" | sudo tee /etc/apt/sources.list.d/mina-unstable.list
sudo apt-get update
Then, install the package(s) that you need:
Mina Daemon: sudo apt-get install -y mina-berkeley=2.0.0rampup1-rampup-68fa3fa
zkApps Test Transaction CLI: sudo apt-get install -y mina-zkapp-test-transaction=2.0.0rampup1-rampup-68fa3fa
[zkApps Transaction CLI Documentation](https://github.com/MinaProtocol/mina/blob/ecbb3d490dce0f199894ebdf787a03b8fec64bd4/src/app/zkapp_test_transaction/README.md)

Docker Images:
Every docker image is built for each of the supported platforms, replace CODENAME in the images below with your choice of bionic, focal, stretch, buster, or bullseye to select the base docker image.
Mina Daemon: gcr.io/o1labs-192920/mina-daemon:2.0.0rampup1-rampup-68fa3fa-CODENAME-berkeley
Mina Archive: gcr.io/o1labs-192920/mina-archive:2.0.0rampup1-rampup--68fa3fa-CODENAME

Step by Step Guide:
This version will only connect and function properly on the newly launched Berkeley Testnet.

Create a libp2p keypair for a node the first time and persist it
mina libp2p generate-keypair -privkey-path <path-to-the-key-file>

Run mina daemon

Set environment variable MINA_LIBP2P_PASS with the password set for the libp2p keypair in step1
Make sure to run the daemon with -libp2p-keypair <path-to-the-key-file> and --peer-list-url https://storage.googleapis.com/seed-lists/berkeley_seeds.txt.
If you are running the correct version on the correct network BERKELEY TESTNET, mina client status will show:

Chain id: 8fab56ef04619087270631e89bde75990d7968309f4008e3cac42eee9601fc96
Git SHA-1:  68fa3faa1af5ab5ee4510ffe9d677a294dd96229

**Comment by garethtdavies** [comment]
Images are not on DockerHub so can get via `gcr.io/o1labs-192920/mina-daemon:<tag>`

**Reply by deepthiskumar** [reply]
Updated the links. Thank you for catching that!
